ml-agents-envs/mlagents_envs/side_channel/side_channel.py

```diff
@@ -39,3 +44,126 @@ def channel_id(self) -> uuid.UUID:
         processed in the environment.
         """
         return self._channel_id
+
+
+class OutgoingMessage:
```

Comment:
I think we should put these in a separate file. The import might look ugly but we could put an `__init__` file in the `side_channel` folder that imports `SideChannel, OutGoingMessage, IncomingMessages`. Thoughts ?

Reply:
Done